frameworks/PHP/laravel/app/Http/Controllers/Controller.php

```diff
@@ -52,7 +52,7 @@ public function updates($queries = 1)
 
         while ($queries--) {
             $row = World::query()->find(self::randomInt());
-            $row->randomNumber = self::randomInt();
+            $row->randomNumber = ($randomInt = self::randomInt()) !== $row->randomNumber ? $randomInt : $randomInt + 1;
```

Comment:
If `$randomInt` is `10000` this would fail, as `10001` is invalid.
Other frameworks generate a new random number instead.